Backend/Backend.iml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
to do .gitignore

Reply:
Nadal ten plik łapie git, więc coś poszło nie tak ;)  Backend/Backend.iml  musi być ignorowane przez gita

Reply:
Ew. mógł być ten plik zcommitowany wcześniej i trzeba go usunąć z indexu i zrobić z tym commit.

Reply:
> Ew. mógł być ten plik zcommitowany wcześniej i trzeba go usunąć z indexu i zrobić z tym commit.

Trzeba było zrobić nowego gitignora i już go nie łapało. Z tego co wyczytałem to zmiany w gitignorze są aktualizowane dla nowych plików.